dali/operators/python_function/dltensor_function.h

```diff
@@ -218,12 +225,23 @@ class DLTensorPythonFunctionImpl : public Operator<Backend> {
     }
   };
 
+  void SetOutputLayouts(workspace_t<Backend> &ws) {
+    if (!layouts_specified_) return;
+    assert(ws.NumOutput() == static_cast<int64_t>(output_layouts_.size()));
```

Comment:
Do you need this assert?
NumOutput is set based on the num_outputs variable, and this is enforced in the constructor.

Reply:
Not really needed.